src/Components/Aspire.Npgsql.EntityFrameworkCore.PostgreSQL/NpgsqlEntityFrameworkCorePostgreSQLSettings.cs

```diff
@@ -8,26 +8,6 @@ namespace Aspire.Npgsql.EntityFrameworkCore.PostgreSQL;
 /// </summary>
 public sealed class NpgsqlEntityFrameworkCorePostgreSQLSettings
 {
-    /// <summary>
-    /// Gets or sets the connection string of the PostgreSQL database to connect to.
-    /// </summary>
-    public string? ConnectionString { get; set; }
-
-    /// <summary>
-    /// Gets or sets a boolean value that indicates whether the DbContext will be pooled or explicitly created every time it's requested.
-    /// </summary>
-    /// <value>
-    /// The default value is <see langword="true"/>.
-    /// </value>
-    /// <remarks>Should be set to false in multi-tenant scenarios.</remarks>
-    public bool DbContextPooling { get; set; } = true;
-
-    /// <summary>
-    /// <para>Gets or sets the maximum number of retry attempts.</para>
-    /// <para>Default value is 6, set it to 0 to disable the retry mechanism.</para>
-    /// </summary>
-    public int MaxRetryCount { get; set; } = 6;
```

Comment:
@adamsitnik @roji - I know we originally set this default max retry when we first added these EF components. Is the default value for EF not good enough for people? Should we try to keep this behavior, or is it unnecessary?

Reply:
I looked back in the git history of the old Aspire repo and I think I did the following here:

It's not enabled by default in EF: https://www.npgsql.org/efcore/misc/other.html#execution-strategy and the users need to call `EnableRetryOnFailure(void)`. 6 is most likely the default value and I wanted to let the users know that this is the value that we use, so I copy-pasted the const here.

Reply:
/cc @AndriySvyryd